dbt/include/trino/macros/apply_grants.sql

```diff
@@ -0,0 +1,29 @@
+{% macro trino__get_show_grant_sql(relation) -%}
+    select
+        grantee,
+        lower(privilege_type) as privilege_type
+    from information_schema.table_privileges
+    where table_catalog = '{{ relation.database }}'
+    and table_schema = '{{ relation.schema }}'
+    and table_name = '{{ relation.identifier }}'
+{%- endmacro %}
+
+{% macro trino__copy_grants() %}
+    {{ return(False) }}
+{% endmacro %}
+
+{%- macro trino__get_grant_sql(relation, privilege, grantees) -%}
+    grant {{ privilege }} on {{ relation }} to {{ adapter.quote(grantees[0]) }}
+{%- endmacro %}
+
+{%- macro trino__support_multiple_grantees_per_dcl_statement() -%}
+    {{ return(False) }}
```

Comment:
Return some information that it's not supported.

Reply:
Addec comment